task/05-regex-tasks.js

```diff
@@ -116,7 +116,8 @@ function getRegexForSSN() {
  *   'Pa55'.match(validator) => false
  */
 function getPasswordValidator(minLength) {
-   throw new Error('Not implemented');
+   let validator = new RegExp(`(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])^[a-zA-Z0-9]{${minLength}}`);
+   return validator; 
```

Comment:
You can return it also in 1 row as:
return new RegExp(`(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])^[a-zA-Z0-9]{${minLength}}`);
:)